### PR TITLE
1.0 release blog

### PR DESCRIPTION
**Pull request by robmarcer**
I still need to add the video link but otherwise this is ready for review please.

### CODE REVIEW

File: src/blog/2022/10/flowforge-1-released.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+<!--more-->
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above. 
+
+We're pleased to announce version 1.0 FlowForge is now available. Keep reading for a promotion code to get your first month free on FlowForge. Version 1.0 represents our vision of the base set of features needed for you to get great value from using FlowForge in a production environment. That's not to say we are done, we will continue to add features, improve our interfaces and fix bugs with the same enthusiasm as we've worked towards 1.0. We'd like to hear your feedback on what we will be including in [1.1 and beyond](https://github.com/orgs/flowforge/projects/5).
+
+## Features
+[Auto assignment of unique IDs to Devices](https://github.com/flowforge/flowforge/issues/841)
+We've updated FlowForge so that each running instance of Node-RED is assigned unique IDs. This can make handling of data returned from multiple [Devices](https://flowforge.com/docs/user/concepts/#device) far more robust.
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:20]:
```suggestion
[Standard Environment Variables set for both Projects and Devices](https://github.com/flowforge/flowforge/issues/841)

Projects now get a set of predefined environment variables that can be used by their flows. These give your flows access to the projects unique id and name. When the flows are deployed to [devices](https://flowforge.com/docs/user/concepts/#device), they also get the device's id and name. That makes is easier to deploy flows across multiple devices and have each able to identify itself.
```

File: src/blog/2022/10/flowforge-1-released.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+[Define custom Nodes to be pre-installed in your Templates](https://github.com/flowforge/flowforge/issues/405)
+This feature allows you to pre-define which custom nodes you want as part of your FlowForge [Templates](https://flowforge.com/docs/user/concepts/#project-template).
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:23]:
```suggestion
[Add additional node modules to your projects](https://github.com/flowforge/flowforge/issues/405)
This feature allows you to pre-define which custom nodes you want to be installed in your FlowForge project, making it easier to manage.
```

File: src/blog/2022/10/flowforge-1-released.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+[Import credentials files into your FlowForge Projects](https://github.com/flowforge/flowforge/issues/835)
+When migrating an existing flow into FlowForge you can now import your credentials file.
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:26]:
```suggestion
[Import existing projects into FlowForge](https://github.com/flowforge/flowforge/issues/835)

You can now import your existing flow and credentials files straight into your FlowForge project - making it really easy to move your existing projects into the platform.
```

File: src/blog/2022/10/flowforge-1-released.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+As an open-source project, we welcome community involvement in what we're building. If you're interested in contributing, checkout our [guide in the docs](https://flowforge.com/docs/contribute/).
+
+### Try it out
+
+[Sign up for FlowForge](https://app.flowforge.com/account/create?code=RELEASE1) with this link  or at the checkout enter the code **RELEASE1** to get your first project free for a month.
+
+### Upgrading FlowForge
+
+[FlowForge](https://app.flowforge.com) is already running 1.0. Upgrade your project Stacks to the latest version to make sure you get all the latest changes.
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:65]:
```suggestion
Our managed [FlowForge](https://app.flowforge.com) is already running 1.0. Upgrade your project Stacks to the latest version to make sure you get all the latest changes.
```

File: src/blog/2022/10/flowforge-1-released.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
+---
+title: FlowForge 1.0 released
+subtitle: FlowForge at 1.0, a huge milestone for us.
+description: FlowForge is now 1.0!
+date: 2022-10-27 18:00:00.0
+authors: ["rob-marcer"]
+video: XXXXXXXXXXXX
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:7]:
```suggestion
video: K4xw09DbSdI
```
Placeholder - use the intro to ff video

File: src/blog/2022/10/flowforge-1-released.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+Auto assignment of unique IDs to Devices, define custom nodes to be pre-installed in your [Templates](https://flowforge.com/docs/user/concepts/#project-template) and import credentials files into your FlowForge [Projects](https://flowforge.com/docs/user/concepts/#project).
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:10]:
```suggestion
Predefined environment variables for your Projects and devices, manage your project's modules and import your existing flows (and credentials!) into your FlowForge [Projects](https://flowforge.com/docs/user/concepts/#project).
```

File: src/blog/2022/10/flowforge-1-released.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+<!--more-->
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above.
```
Comment (knolleary) [inline src/blog/2022/10/flowforge-1-released.md:14]:
```suggestion
<!-- Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above.  -->
```

We can restore this when swapping in the 1.0 release video

**Review by knolleary — COMMENTED** [review]
Suggested changes on how we describe these headline features. My apologies for not engaging on this earlier

File: src/blog/2022/10/flowforge-1-released.md (hunk 6 above)
Comment (Steve-Mcl) [reply src/blog/2022/10/flowforge-1-released.md:10]:
```suggestion
Predefined environment variables for your Projects and Devices, manage your Project's modules and import your existing flows (and credentials) into your FlowForge [Projects](https://flowforge.com/docs/user/concepts/#project).
```

File: src/blog/2022/10/flowforge-1-released.md (hunk 2 above)
Comment (Steve-Mcl) [reply src/blog/2022/10/flowforge-1-released.md:24]:
```suggestion
[Define additional Nodes to be pre-installed](https://github.com/flowforge/flowforge/issues/405)
This feature allows you to pre-define additional Node-RED nodes and node modules you may want as part of your FlowForge [Templates](https://flowforge.com/docs/user/concepts/#project-template).
```

Comment (knolleary) [reply src/blog/2022/10/flowforge-1-released.md:24]: Given they are not part of the admin-created template, I'm wary of keeping saying template. I prefer my version.

Comment (Steve-Mcl) [reply src/blog/2022/10/flowforge-1-released.md:24]: I made a booboo

My suggestion was supposed to remove the "templates" and "custom" nodes parts.  Custom suggests something other than an existing contrib or NPM module.

Comment (Steve-Mcl) [reply src/blog/2022/10/flowforge-1-released.md:24]:
```suggestion
[Add additional node modules to your projects](https://github.com/flowforge/flowforge/issues/405)
This feature allows you to pre-define additional Node-RED nodes and node modules you may want to be installed in your FlowForge project, making it easier to manage.
```

Comment (Steve-Mcl) [reply src/blog/2022/10/flowforge-1-released.md:24]: that was what i meant to write DOH

**Pull request merged**